Jenkinsfile

```diff
@@ -1,13 +1,13 @@
-def comonVersion = "2.356"
-def recentLTS = "2.346.1"
+def latestSupported = "2.453"
+def recentLTS = "2.361.4"
 def configurations = [
-    [ platform: "linux", jdk: "8", jenkins: null ],
+    [ platform: "linux", jdk: "11", jenkins: null ],
     // windows 
-    [ platform: "windows", jdk: "8", jenkins: comonVersion ],
+    [ platform: "windows", jdk: "17", jenkins: latestSupported ],
     // java 11 
-    [ platform: "linux", jdk: "11", jenkins: comonVersion ],
+    [ platform: "linux", jdk: "17", jenkins: latestSupported ],
     // windows
-    [ platform: "windows", jdk: "8", jenkins: recentLTS ],
+    [ platform: "windows", jdk: "11", jenkins: recentLTS ],
     // java 11
```

Comment:
minor nitpick: the comment needs to be updated (either "Java 17" or something about the latest supported version)